start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Start tabpy
+tabpy &
+
+# Wait for tabpy server to bootup
```

Comment:
Anyway we can check if tabpy is actually running? That will be more certain than sleep

Reply:
I thought about this earlier, but it might get unnecessarily complicated.
We can poll on /info api until we get 200 response. To avoid waiting indefinitely, we can add a max retry limit and throw an error I suppose. We also need to know whats the tabpy port number while polling.